### PR TITLE
Fix diagram, do imply ID docs inserted into TS

### PR DESCRIPTION
**Pull request by aj-stein-nist**
Per discussion in today's editor meeting, fix the error in the diagram.

### CODE REVIEW

File: draft-ietf-scitt-architecture.md [hunk 1]
```diff
@@ -339,9 +339,9 @@ Most of the details of the Receipt's contents are specified in the COSE Signed M
               v                   '------+----'     |
          .----+----.                     |          |
         |  Signed   |    COSE Signing    |          |
-        | Statement +<-------------------+          |
-         '----+----'                     |          |
-              |                 +--------+------+   |
+        | Statement +<------------------'          |
```
Comment (aj-stein-nist) [inline draft-ietf-scitt-architecture.md:342]:
```suggestion
        | Statement +<------------------'           |
```

**Review by OR13 — APPROVED** [review]
This PR should be merged.

**Comment by SteveLasker** [comment]
From the discussion today, the "Identity Documents" box refers to:

**A.** Generally represent identity documents used for signing and verifying (private and public keys)
**B.** Only represent the public key for verification
**C.** Only represent the private key for signing

**The Original drawing** represented A, but was missing an arrow into the Transparency Service, leaving interpretation there might be outbound content:
image

1. Sign the statement with a private key
2. Both verify the signed statement using the public key from 1 above, and sign the receipt with a private key from a different identity document for the transparency service
3. Verify the signed statement, or the receipt, with the public key of the identity
4. Unsure what this represents as it seems duplicative with 1

**This PR** resolves the directionality concern raised in the meeting, but didn't resolve the inconsistent use of identity documents representing the private and public keys:

image

1. Sign the statement with a private key
2. Removes the directionality, but removes the reference to the transparency service having the public key for verification
3. Maintain the reference to verifying the signed statement, or the receipt, with the public key of the identity - Inconsistent with 2 above
4. Unsure what this represents as it seems duplicative with 1

If we use **Identity Documents to represent Public Keys for verification only**, would this be the accurate image:

image

1. -N/A
2. Verify the signed statement using the public key from 1 above, although we removed the line for 1
3. Maintain the reference to verifying the signed statement, or the receipt, with the public key of the identity 
4. Unsure what this represents as it seems duplicative with 1

Suggestion to have Identity Documents represent the private and public keys used to represent an Identity, showing how Identity is key to signing and verification steps

image

1. Sign the statement with a private key
2. Both verify the signed statement using the public key from 1 above, and sign the receipt with a private key from a different identity document for the transparency service
3. Verify the signed statement, or the receipt, with the public key of the identity
4. removed as it was duplicative with 1

To elaborate a bit further, I've made PR#240 to represent the above, with a little more clarity the box of identity documents represents the ability to sign or verify

image

**Review by SteveLasker — CHANGES_REQUESTED** [review]
This solves the directional flow issue raised in the editors meeting, but created inconsistency for using the identity documents for signing and verification.

See [comment on this PR](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/239#issuecomment-2110755927), and PR #240 for the suggestion

**Comment by aj-stein-nist** [comment]
I saw the analysis, I am not sure I agree with the conclusions, but I am not going to hold up the WG effort on the draft over this. It is a minor detail. I am closing this now in favor of #240 anyway since this will not get merged. Thanks for the explanation and background.

File: draft-ietf-scitt-architecture.md [hunk 2]
```diff
@@ -339,9 +339,9 @@ Most of the details of the Receipt's contents are specified in the COSE Signed M
               v                   '------+----'     |
          .----+----.                     |          |
         |  Signed   |    COSE Signing    |          |
-        | Statement +<-------------------+          |
-         '----+----'                     |          |
-              |                 +--------+------+   |
+        | Statement +<------------------'           |
```
Comment (SteveLasker) [inline draft-ietf-scitt-architecture.md:342]: The other suggestion is to simply add directionality at the TS, while maintaining the curve to the signed statement
```suggestion
        | Statement +<------------------'|          |
```
image

Which also needs the suggestion in line 343

**Pull request closed without merging**